### PR TITLE
Adding contents to about section of cynergy v2

### PR DESCRIPTION
**Pull request by sadhana1058**
I  have 2 things left.I have to make the 2 lines with about  heading  responsive and adjust image in such a way it does not overlap the text. Apart from that if there are any changes @mittalchirag bhaiya do let me know

### CODE REVIEW

**Review by mittalchirag — COMMENTED** [review]
@sadhana1058 There are some gibberish files in your PR. Please clean your branch and update the PR.

**Comment by sadhana1058** [comment]
I have removed those unwanted files. Have a look at it @mittalchirag

File: src/components/sections/about.js [hunk 1]
```diff
@@ -1,20 +1,112 @@
 import React from 'react';
 import styled from 'styled-components';
-import { Section } from '@design/theme';
 import { Heading } from '@components';
+import { Section, ts36r, ts20r, ts29l } from '@design/theme';
+import { media } from '@design/media';
+import photo from '../../images/about.png';
```
Comment (mittalchirag) [inline src/components/sections/about.js:6]: No need to import here as you have added in the Images module.

File: src/components/sections/about.js [hunk 2]
```diff
@@ -1,20 +1,112 @@
...
+            <AboutBody>
+                <p>
+                    Cynergy is the first of its kind coding club at <Co>RUAS</Co>. The purpose of Cynergy is embodied in its core open principles - <Co>Collaboration</Co>, <Co>Innovation</Co> and <Co>Competition</Co>. Cynergy is a platorm to promote learning and development of programming skills. It empowers students
+                    with competitive programming skills through activities like Hackathon and Share sessions.
+                </p>
+            </AboutBody>
+
+            <img
+                src={photo}
+                style={{
```
Comment (mittalchirag) [inline src/components/sections/about.js:96]: Avoid Inline css

File: src/components/sections/about.js [hunk 3]
```diff
@@ -1,20 +1,112 @@
...
+        margin-bottom: 10%;
+        margin-left:auto;
+        margin-right:10px;
+    `}
+`;
 const About = () => {
     return (
         <AboutContainer>
-            <Heading orientation="vertical">about</Heading>
+            <Heading orientation="vertical" maxLength1="290px" maxLength2="880px">
```
Comment (mittalchirag) [inline src/components/sections/about.js:84]: Avoid hardcoding line length in pixels. Use vh or vw.

File: src/components/heading.js [hunk 4]
```diff
@@ -13,20 +13,20 @@ const HeadingContainer = styled.div`
...
     transform: rotate(-180deg);
     direction: rtl;
 `;
 
-const Heading = ({ orientation, children }) => {
+const Heading = ({ orientation, children, maxLength1, maxLength2 }) => {
     return (
         <HeadingContainer>
-            <div className="vertical">
+            <div className="vertical" style={{ height: '325px', width: '10px' }}>
```
Comment (mittalchirag) [inline src/components/heading.js:25]: Why inline css here?

**Review by mittalchirag — APPROVED** [review]
Overall good work but a bit of complex and unnecessary CSS.

**Pull request merged**